h/schemas/annotation.py

```diff
@@ -311,3 +311,82 @@ def _target_selectors(targets):
         return targets[0]['selector']
     else:
         return []
+
+
+class SearchParamsSchema(JSONSchema):
+    """Validates a JSON payload for an elasticsearch query via API"""
```

Comment:
This description needs updating. The schema validates query parameters to an `/api/search` request, not the JSON query that is sent to Elasticsearch - that's an implementation detail of the search API.